apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx

```diff
@@ -126,6 +143,34 @@ const NftViewer = ({ route }: Props) => {
 
   const nftCapabilities = useMemo(() => getNftCapabilities(nft), [nft]);
 
+  const [floorPriceLoading, setFloorPriceLoading] = useState(false);
+  const [ticker, setTicker] = useState("");
+  const [floorPrice, setFloorPrice] = useState(null);
+
+  useEffect(() => {
+    let mounted = true
+    if(mounted) {
```

Comment:
this is always true. useEffect function is never called again after unmounted.

I think instead you meant to do the if in the `.then(` success and error callbacks.